src/app/app.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpClientModule } from '@angular/common/http';
+
+import { AppRoutingModule } from './app-routing.module';
+
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatTableModule } from '@angular/material/table';
+import { MatSortModule } from '@angular/material/sort';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatProgressBarModule } from '@angular/material/progress-bar';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { MatMenuModule } from '@angular/material/menu';
+
+import { AppComponent } from './app.component';
+import { FilterComponent } from './filter/filter.component';
+import { FormsModule } from '@angular/forms';
+import { LegendComponent } from './legend/legend.component';
```

Comment:
minor: try to type `Alt+Shift+O` while you are in this file

Reply:
Wow, I'll do it always from now on 😮